### PR TITLE
Issue56 - report call when connected

### PR DESCRIPTION
**Pull request by bchen-twilio**
#56 

Use the `reportOutgoingCallWithUUID:connectedAtDate:` method to notify the (CallKit) system that the call is now connected otherwise the system will treat it as a canceled call even if the call was accepted and connected.

### CODE REVIEW

**Comment by ceaglest** [comment]
The same comments from the Objective-C PR apply here. Lets figure out what we want to do and get these merged. https://github.com/twilio/voice-quickstart-objc/pull/54

File: SwiftVoiceCallKitQuickstart/ViewController.swift [hunk 1]
```diff
@@ -193,6 +194,7 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
         NSLog("callDidConnect:")
         
         self.call = call
+        self.callKitCompletionCallback!(true)
```
Comment (ceaglest) [inline SwiftVoiceCallKitQuickstart/ViewController.swift]: Minor - you could also clear `self.callKitCompletionCallback!` here and in the failure case.

File: SwiftVoiceCallKitQuickstart/ViewController.swift [hunk 2]
```diff
@@ -328,15 +326,13 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
...
         // TwilioVoice.sharedInstance().configureAudioSession()
-
-        guard let call = self.callInvite?.accept(with: self) else {
-            action.fail()
-            return
+        
+        self.performAnswerVoiceCall(uuid: action.callUUID) { (success) in
+            if (!success) {
+                action.fail()
+            }
```
Comment (ceaglest) [inline SwiftVoiceCallKitQuickstart/ViewController.swift]: Fulfill the action if successful.

File: SwiftVoiceCallKitQuickstart/ViewController.swift [hunk 3]
```diff
@@ -193,6 +194,8 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
         NSLog("callDidConnect:")
         
         self.call = call
+        self.callKitCompletionCallback!(true)
+        self.callKitCompletionCallback = nil
```
Comment (ptankTwilio) [inline SwiftVoiceCallKitQuickstart/ViewController.swift]: 👍

**Pull request merged**